spec/amp-var-substitutions.md

```diff
@@ -1374,3 +1375,16 @@ Find the specified (or first) matched pattern.
 ##### Equals
 
 -   Example: `$EQUALS(${val}, value)` resolves to `'true'` or `'false'`
+
+##### Simple Calculations
+
+This macro allows you to perform basic arithmetics like addition, subtraction, multiplication and division.
+
+Parameters:
+
+-   **leftOperand** : A variable or a literal that resolves to a number or, a number string.
+-   **rightOperand** : A variable or a literal that resolves to a number or, a number string.
+-   **operation** : A literal. One of ADD, SUBTRACT, MULTIPLY, DIVIDE
```

Comment:
I'm not sure I like using an uppercase version of these strings to signify what calculation to run, since uppercase values are also what triggers macros. What do you think about using `+, -, /, *`?

Reply:
My initial thought on using strings instead of symbols is to reduce ambiguity. For example, `+` in javascript can do addition as well as string concatenation. Similarly, `-` is used to negate the sign of the number. 

Furthermore, I initially wanted to suggest a lower case string instead. But on second thought, We could save a few chars on the final js if we go for symbols. Just need a bit more documentation. Shall I proceed with the change?

Reply:
@micajuine-ho Also, if we are going with the operator symbols, should we change the order of the parameters as well? Like operators between the operands for better readability? What is your thought on this?

Reply:
Agreed. I'll make the changes.